### DISCUSSION TITLE
How can I mark items as "registered"?

### DISCUSSION THREAD

**Discussion by tscpp** [jotaen/klog#238] [Help / Question] [answered]
I am using an external website to register my time. I simply copy and paste the record from my editor to the website and enter the date. Seems simply enough, however I managed once again to create duplicate records (yes it is possible) on the website.

I was thinking if there is a way to mark records as complete/registered, or comment where I most recently registered the time on my external tool. #54 is a possible solution to this.

I have tried using tags and a bunch of newlines to separate registered and not-registered records. Neither of the two works well.

**Comment by jotaen** [comment, marked as answer]
Apart from the ideas that you mentioned (blank lines, tags), I also can’t think of anything in the file format to solve this use-case.

How often do you copy over your data from your PC to that website? In case you do this once e.g. at the end of a day or the end of a week, you could maybe use `klog print --today` or `klog print --this-week`, so that you only see the relevant records on the terminal for copying.

I think in the ideal case, the external tool should be in charge of filtering out the data it already knows. So e.g., if you have one file per month, then you’d just upload the entire file, and the website would disregard/update/prune its own state. But of course that might not be viable for you, unless you are in control of that website (which I assume you aren’t).
